db/migrate/20161215161941_add_enabled_field_to_ems.rb

```diff
@@ -0,0 +1,5 @@
+class AddDisabledFieldToEms < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :enabled, :boolean, :null => false, :default => true
```

Comment:
I kept getting `enabled = nil` on newly created providers though im not sure this is the way to go. 
This migration needs to 
1) make all existing providers have `enabled = true`
2) make all new providers have `enabled = true`